ucsb-cs156-s22/team02-s22-5pm-2
# Ryan article

# Overview

This is a general pull request that brings over Controller, entity, and tests for the Article service

# Issues Addressed

#2 #3 #4 #5 

# Test Plan (for interactive testing)

Replace this sentence with a step by step list of what actions to take to demonstrate that this PR works successfully.
* Run app locally, go to swagger and test out each functionality of the controller
* Open the heroku app and do the same ensuring functionality
* Run mvn tests to ensure unit tests are working
